README.md

```diff
@@ -111,6 +111,32 @@ cd compiler
 ./fetch.sh # If you just want the current playground
 ```
 
+### Troubleshooting
+
+#### Mac
+
+After launching `ui` successfully locally, when you try to do any action (ex. `build`, `rustfmt`, `run` and so on), if you get errors about Docker Mount denied like following
+
+```
+docker: Error response from daemon: Mounts denied:
+The paths /var/folders/dx/l5pn75zx5v9_cwstvgwc5qyc0000gn/T/playground.6gEHdGUM6XPU/output and /var/folders/dx/l5pn75zx5v9_cwstvgwc5qyc0000gn/T/playground.6gEHdGUM6XPU/input.rs
+are not shared from OS X and are not known to Docker.
+You can configure shared paths from Docker -> Preferences... -> File Sharing.
+See https://docs.docker.com/docker-for-mac/osxfs/#namespaces for more info.
+.
+time="2099-12-31T00:00:00+00:00" level=error msg="error waiting for container: context canceled"
+```
+
+Then, you can set `TMPDIR` to fix this issue.
```

Comment:
Presumably you also need to configure Docker to include the path to the new `TMPDIR` in the set of directories it can use for mounting, as alluded to in the error message?

Reply:
Yeah, you're right. If the user set a `TMPDIR` which is not included in the Docker preference, then he/she will need to set it.

I didn't need to set it because I set the `TMPDIR` under my project path which is under `/Users` and `/Users` is also a default setting in Docker for Mac.